packages/dev/loaders/src/BVH/bvhFileLoader.ts

```diff
@@ -82,7 +82,7 @@ export class BVHFileLoader implements ISceneLoaderPluginAsync, ISceneLoaderPlugi
                 geometries: [],
                 lights: [],
                 spriteManagers: [],
-            });
+            } as ISceneLoaderAsyncResult);
```

Comment:
Why the `as`? Is it just for intellisense? If so it seems like `satisfies` would make more sense here since the intent is not to cast.